python-src/resources.py

```diff
@@ -39,7 +39,8 @@ def post(self):
         game = Game()
         request_data = request.get_json()
         game.players = [ndb.Key(urlsafe=request_data['player_id'])]
-        game.cities = Util.get_cities(int(request_data.get('number_of_cities', 3)))
+        game.cities = Util.get_cities(int(request_data.get('number_of_cities', 3)),
```

Comment:
I tried to add a diff property to Games in backend and it was not success. I tried game.diff = request_data.get('difficulty', 0) and game.diff = request_data['difficulty'] but the command prompt said they were both wrong. I did this because in highscore component, the start a new game button leads to a new room and create game needs game difficulty. It might be good if the difficulty is the same as last game. Otherwise the difficulty has to be added in url